data/pg/pucar/DocumentType.json

```diff
@@ -0,0 +1,160 @@
+{
+  "tenantId": "pg",
+  "moduleName": "pucar",
+  "DocumentType": [
+    {
+      "code": "AADHAAR CARD",
+      "active": true
+    },
+    {
+      "code": "PAN CARD",
```

Comment:
@manimaarans Define code with underscores. Don't have spaces in the code.

Reply:
@manimaarans - if you are using MDMS v2, no need to define an ID. It will autogenerate one. But v1, requires id definition. Then you need to migrate this to V2. Please keep track of this.